per/serializers.py

```diff
@@ -1253,3 +1253,36 @@ class Meta:
             "id",
             "title",
         ]
+
+
+class LearningByRegionSerializer(serializers.Serializer):
+    region_name = serializers.CharField()
+    count = serializers.IntegerField()
+
+
+class LearningByCountrySerializer(serializers.Serializer):
+    country_id = serializers.IntegerField()
+    country_name = serializers.CharField()
+    count = serializers.IntegerField()
+
+
+class LearningBySectorSerializer(serializers.Serializer):
+    title = serializers.CharField()
+    count = serializers.IntegerField()
+
+
+class LearningSourcesOvertimeSerializer(serializers.Serializer):
+    type = serializers.IntegerField()
```

Comment:
@susilnem Do we return type as integer or get display name for type? If no may be we need to do `type = serializers.CharField(source='get_type_display', read_only=True)` to get the type name.

Reply:
No, for now it's not required.

Reply:
Are we sure about this? It will be complicated for the client side to do this mapping.

`source='get_type_display'` can't be used here, as the object will be a dict instead of a proper object. We could use serializer_method or do the enum->label in the views.